src/proto/grpc/testing/control.proto

```diff
@@ -92,28 +73,82 @@ message SecurityParams {
 message ClientConfig {
   // List of targets to connect to. At least one target needs to be specified.
   repeated string server_targets = 1;
-  ClientType client_type = 2;
```

Comment:
I am not convinced that enum as a client type is wrong. If we need some language-specific client types (or server types) we can just add them to the enumeration and it will only add to the transparency. do we already know what language-specific client types we are expecting?